CHANGELOG.md

```diff
@@ -1,5 +1,28 @@
 # Changelog
 
+## 1.0.0 (March 8, 2017)
+
+- Switched amalgam8 sidecar to use Envoy proxy from Lyft (https://github.com/lyft/envoy)
+
+- Support for HTTP/2, gRPC proxying (downstream and upstream)
+
+- Support for gRPC over HTTP/1.1
```

Comment:
IIRC. Envoy also supports protocol bridging between gRPC on 1.1 and 2.0. Please confirm with @rshriram